package.json

```diff
@@ -92,6 +92,7 @@
     "node": ">=12"
   },
   "dependencies": {
+    "@aws-sdk/lib-dynamodb": "^3.170.0",
```

Comment:
Was this a mistake? I don't see any changes at the root level

Reply:
Might be my lack of understanding of lerna, but to get the dependency into this subproject can we just reference it in the parent package.json and then lerna will pull it into the subproject?

Reply:
Well hold on, my understanding is that lerna can "hoist" built node_modules from the package level to the root level, so you don't have duplicate installations. But I thought the idea was still to define each dependency needed in the package.json of the package itself. Let me know if you see differently in the docs though!